Packs/EWS/Integrations/EWSv2/EWSv2.py

```diff
@@ -2125,7 +2125,14 @@ def sub_main():
 
             error_message_simple += "You can try using 'domain\\username' as username for authentication. " \
                 if AUTH_METHOD_STR.lower() == 'ntlm' else ''
-        if "Status code: 503" in debug_log:
+
+        if "SSL: CERTIFICATE_VERIFY_FAILED" in debug_log:
+            # same status code (503) but different error.
+            error_message_simple = "Got timeout from the server. " \
```

Comment:
Remove this line about timeout. As it is not timeout related. Let's use the following message:

`Certificate verification failed - This error may happen if the server certificate cannot be validated or as a result of a proxy that is doing SSL/TLS termination. It is possible to bypass certificate validation by checking 'Trust any certificate' in the instance settings.`

@kirbles19 please review this message.